source/content/guides/secrets/01-introduction.md

```diff
@@ -0,0 +1,42 @@
+---
+title: Pantheon Secrets Guide
+subtitle: Introduction
+description: Securely store secrets in the Pantheon Platform.
+type: terminuspage
+contributors: [stovak]
+contenttype: [guide]
+innav: [true]
+categories: [secrets]
+cms: [drupal, wordpress]
+audience: [development]
+product: [secrets]
+integration: [--]
+tags: [reference, cli, local, terminus, workflow]
+permalink: docs/guides/secrets
+reviewed: "2024-05-01"
+---
+
+Pantheon Secrets is key to maintaining industry best practices for secure builds and application implementation. This feature provides a convenient mechanism for you to manage your secrets and API keys directly on the Pantheon platform.
+This guide covers features and use cases of the Pantheon Secrets feature; it could also be referred as Secrets Manager because that is the Terminus plugin name.
+Features
+
+* Secure: secrets are encrypted at rest.
+* Easy to use: create and update secrets via Terminus.
+* Governability: secrets could be set at organization level and shared to all the sites owned by that organization.
```

Comment:
"Governability" doesn't align with the other lines. "Governable" would be consistent.